jwst/datamodels/schemas/ifuimage.schema.yaml

```diff
@@ -0,0 +1,41 @@
+allOf:
+- $ref: core.schema.yaml
+- $ref: bunit.schema.yaml
+- type: object
+  properties:
+    data:
+      title: The science data
+      fits_hdu: SCI
+      default: 0.0
+      ndim: 2
+      datatype: float32
+    dq:
+      title: Data quality array
+      fits_hdu: DQ
+      default: 0
+      datatype: uint32
+    err:
+      title: Error array
+      fits_hdu: ERR
+      default: 0.0
+      datatype: float32
+    zeroframe:
+      title: Zeroframe array
+      fits_hdu: ZEROFRAME
+      default: 0.0
```

Comment:
I wonder if we should remove the default values for all arrays that are optional or don't appear in all datasets, so that the extension is not written out to the product when it's not needed for a given product. This would include zeroframe, area, relsens2d, and all of the ones in pathloss.schema.